lib/utilities/monitoringHandler.js

```diff
@@ -3,65 +3,65 @@ const client = require('prom-client');
 
 const collectDefaultMetrics = client.collectDefaultMetrics;
 const numberOfBuckets = new client.Gauge({
-    name: 'cloud_server_number_of_buckets',
+    name: 's3_cloudserver_buckets',
     help: 'Total number of buckets',
 });
 const numberOfObjects = new client.Gauge({
-    name: 'cloud_server_number_of_objects',
+    name: 's3_cloudserver_objects',
```

Comment:
should have a unit : `..._objects_count`

Reply:
same as above.